pennylane/labs/vibrational_ham/localize_modes.py

```diff
@@ -0,0 +1,206 @@
+"""This module contains functions to localize normal modes."""
+
+import warnings
+import numpy as np
+import scipy
+
+au_to_cm = 219475
+hbar = 6.022 * 1.055e12  # (amu)*(angstrom^2/s)
+c_light = 3 * 10**8  # m/s
+
+# pylint: disable=dangerous-default-value, too-many-statements
+
+
+def _pm_cost(q):
+    r"""Pipek-Mezey cost function whose minimization yields localized displacements."""
+    nnuc, _, nmodes = q.shape
+
+    xi_pm = 0.0
+
+    for p in range(nmodes):
+        for i in range(nnuc):
+            q2 = 0.0
+            for alpha in range(3):
+                q2 += q[i, alpha, p] ** 2
```

Comment:
```suggestion
q2 = sum(q[i, alpha, p] ** 2 for alpha in range(3))
```

You could also do the entire loop in a one liner but to me it's kind of hard to read
```suggestion
            xi_pm += sum((sum(q[i, alpha, p] ** 2 for alpha in range(3)))**2 for i in range(nnuc) for p in range(nmodes))
```